### DISCUSSION TITLE
Example of resolutions with OBS/WHIP

### DISCUSSION THREAD

**Discussion by irg1008** [bluenviron/mediamtx#3508] [Questions]
### Question

Could you provide and example on how to create multiple resolution HLS streaming with master playlist for a WHIP server streaming from OBS. I saw you have an example of remuxing for paths but I don't know how to apply it to the obs streaming using onReady hook. Thank you and sorry if it comes across asa newby question, I AM a newby in the streaming world. Thanks again

**Comment by aler9** [comment]
this is currenty unimplemented but is being tracked here: https://github.com/bluenviron/mediamtx/issues/1948

[a comment by an automated account is not shown]
